src/std/10-std-builder.md

```diff
@@ -1 +1,175 @@
 # Builder
+
+The **builder** module defines a `builder` type that allows to build and install C++ modules to be used with Feral
+
+A guide on how to write and build modules for Feral can be read [here](https://levelup.gitconnected.com/writing-c-modules-for-feral-391c30ac7739).
+
+All the given examples assume that the **builder** module was imported using:
+```
+let builder = import('std/builder');
+let io = import('std/io');
+let sys = import('std/sys');
+```
+
+## Functions
+- [new](#new)
+
+## `builder` Member Functions
+- [make_dll](#make_dll)
+- [add_comp_opts](#add_comp_opts)
+- [add_inc](#add_inc)
+- [add_lib](#add_lib)
+- [add_src](#add_src)
+- [perform](#perform)
+
+### new
+```
+builder.new() -> builder
+```
+Creates a new builder with default build settings
+
+Example:
+```
+let build = builder.new();
+io.println(build);
+```
+
+Possible output:
+```
+typeid<14>{linker_flags:  , lib_flags:  -lferalvm -lgmpxx -lgmp -lmpfr , lib_dirs:  -L/usr/lib -L/usr/local/lib/feral , srcs:  , ccache: /usr/bin/ccache , compiler: g++, is_dll: false, compiler_opts:  -std=c++11 -O2 , inc_dirs:  -I/usr/include }
```

Comment:
in commit 50217df of Feral-Std, added typename for builder_t so now it will show `builder_t` instead of `typeid<14>`